packages/graphviz.rb

```diff
@@ -0,0 +1,27 @@
+require 'package'
+
+class Graphviz < Package
+  description 'Graphviz is open source graph visualization software.'
+  homepage 'https://www.graphviz.org/'
+  version '2.40.1'
+  source_url 'https://gitlab.com/graphviz/graphviz/repository/67cd2e5121379a38e0801cc05cce5033f8a2a609/archive.tar.bz2'
+  source_sha256 '8dddc80b4194b17c1a5bc1cc7b5e001e9a3ec27272287dc16f278c736a29a9b9'
+
+  depends_on 'automake' => :build
+  depends_on 'fontconfig'
```

Comment:
We can remove fontconfig, right?

Reply:
Why? I have not tested it without fontconfig. I follow the suggested depedencies from website.

Reply:
I already explained the reason.  It's because pango and poppler include cairo as a dependency and fontconfig is a dependency of cairo.

Reply:
So what this means basically is you don't need to include it since it will be included by default from the other packages.

Reply:
As a matter of fact, you don't even need pango since it's already a dependency of gtk_pixbuf.